dissect/cstruct/parser.py

```diff
@@ -324,6 +325,13 @@ def _names(self, tokens: TokenConsumer) -> list[str]:
 
         return names
 
+    def _include(self, tokens: TokenConsumer) -> None:
+        include = tokens.consume()
+        pattern = self.TOK.patterns[self.TOK.INCLUDE]
+        match = pattern.match(include.value).groupdict()
+
+        self.cstruct.includes.append(match["name"].strip().strip('"'))
```

Comment:
```suggestion
        self.cstruct.includes.append(match["name"].strip().strip("'\""))
```
So we also strip single quotes.

Reply:
[4319927](https://github.com/fox-it/dissect.cstruct/pull/129/commits/4319927a7200704987ec1b31375b41acc7d3f75e)

Reply:
Thanks for commenting :)